charts/datadog/templates/_daemonset-volumes-linux.yaml

```diff
@@ -160,6 +160,9 @@
 - hostPath:
     path: /var/lib/docker
   name: host-docker-dir
+- hostPath:
+    path: /var/lib/containers
+  name: host-crio-dir
```

Comment:
i'm wondering this change will not break GKE autopilot

Reply:
Upon double checking, this change does break GKE autopilot if any of the `datadog.sbom.*` features enabled.

Reply:
This PR should prevent the new hostPath from breaking autopilot: https://github.com/DataDog/helm-charts/pull/1666/files

Reply:
`Upon double checking, this change does break GKE autopilot if any of the datadog.sbom.* features enabled.`

This is even without this change, right ?